src/parser/jobs/mnk/modules/RiddleOfWind.tsx

```diff
@@ -0,0 +1,84 @@
+import {t} from '@lingui/macro'
+import {Trans} from '@lingui/react'
+import {ActionLink} from 'components/ui/DbLink'
+import {ActionRoot} from 'data/ACTIONS'
+import {dependency} from 'parser/core/Injectable'
+import {BuffWindow, EvaluatedAction, WindowEvaluator} from 'parser/core/modules/ActionWindow'
+import {HistoryEntry} from 'parser/core/modules/ActionWindow/History'
+import {GlobalCooldown} from 'parser/core/modules/GlobalCooldown'
+import {SimpleStatistic, Statistics} from 'parser/core/modules/Statistics'
+import React from 'react'
+import {DISPLAY_ORDER} from './DISPLAY_ORDER'
+
+/**
+ * While it is possible to get 16 attacks in a Riddle of Wind
+ * window, it is such a niche and time based thing that I'm
+ * not sure if it's worth it. I've looked into a lot of top
+ * parsing monk logs, and it almost feels like it's RNG if
+ * someone has a 16 RoW attack window.
+ */
+const MAX_ATTACKS_PER_ROW_WINDOW = 15
```

Comment:
No need to limit width so much for the comment, could rename the const to something like `EXPECTED_AUTOS_PER_WINDOW` since it's expected, not max.